.github/workflows/go.yml

```diff
@@ -38,7 +38,30 @@ jobs:
       env:
         CGO_ENABLED: 1
       run: go test -cpu="1,4" -short -race -v ./...
-      
+
+  generate:
+    strategy:
+      matrix:
+        pkg: [s2, zstd]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
```

Comment:
Deliberately running this under 1.17 because of the build constraint behavior. Currently `avo` decides how to output them entirely based on the Go version it was compiled with. So under Go 1.18 there are only `go:build` lines, not `+build`. Since the above job checks Go 1.16 I assumed you were trying to keep 1.16 compatibility for now?

https://github.com/mmcloughlin/avo/issues/183#issuecomment-955164201

Reply:
Correct. I keep 3 versions of backcompat.